lib/src/connector/ischool_plus_connector.dart

```diff
@@ -58,13 +65,32 @@ class ISchoolPlusConnector {
         final value = node.attributes['value'];
         data[name] = value;
       }
+
+      // Step 2
+      // The action should be "oauth2Server.do".
+      // If not, the website is changed and need to fix.
+      // TODO: Add the validation to check Step 1 is died. (It should not die if auth is correct)
       final jumpUrl = tagNode.getElementsByTagName("form")[0].attributes["action"];
-      parameter = ConnectorParameter(jumpUrl);
+      parameter = ConnectorParameter("${NTUTConnector.host}$jumpUrl");
       parameter.data = data;
 
       Response<dynamic> jumpResult = (await Connector.getDataByPostResponse(parameter));
+      tagNode = html.parse(jumpResult.data.toString().trim());
+      nodes = tagNode.getElementsByTagName("a");
+      final redirectUrl = nodes.first.attributes["href"];
+
+      // Step 3
+      // The redirectUrl is provided by <a> HTML DOM on Step 2.
+      // It should be https://istudy.ntut.edu.tw/login2.php with lot of the parameters.
+      parameter = ConnectorParameter(redirectUrl);
```

Comment:
```suggestion
      // Step 3
      // The redirectUrl is provided by <a> HTML tag on Step 2.
      // It should be "https://istudy.ntut.edu.tw/login2.php" with lot of the parameters.
      final redirectUrl = nodes.first.attributes["href"];

      parameter = ConnectorParameter(redirectUrl);
```

Suggest moving this comment to `redirectUrl`, since you are describing it.

Reply:
Fix on 6458cdd